python/tank/util/metrics.py

```diff
@@ -180,16 +180,21 @@ def workers(self):
         """A list of workers threads dispatching metrics from the queue."""
         return self._workers
 
-    def _metrics_supported(self, sg_connection):
+    def _metrics_supported(self, tk):
         """Returns True if server supports the metrics api endpoint."""
 
         if not hasattr(self, '_metrics_ok'):
 
-            self._metrics_ok = (
-                hasattr(sg_connection, 'server_caps') and
-                sg_connection.server_caps.version and
-                sg_connection.server_caps.version >= (6, 3, 11)
-            )
+            from tank.api import get_authenticated_user
```

Comment:
Why a local import? Is it because there is a circular dependency otherwise? If that's the case, could you say so in the code comments? We generally try to avoid local imports and put such comments when circular imports causes issues.